### PR TITLE
Add Umbraco Heartcore blog example

### PR DESCRIPTION
**Pull request by rasmusjp**

### CODE REVIEW

**Comment by ijjk** [comment]
# Stats from current PR

<details>
<summary><strong>Default Server Mode</strong> (Increase detected ⚠️)</summary>

<br/>

<details>
<summary><strong>General</strong></summary>

|  | vercel/next.js canary  | rasmusjp/next.js add-umbraco-heartcore-blog-example | Change |
| - | - | - | - |
| buildDuration | 10.4s | 10.4s | ⚠️ +4ms |
| nodeModulesSize | 73.3 MB | 73.3 MB | ✓ |

</details>

<details>
<summary><strong>Page Load Tests</strong> Overall increase ✓</summary>

|  | vercel/next.js canary  | rasmusjp/next.js add-umbraco-heartcore-blog-example | Change |
| - | - | - | - |
| / failed reqs | 0 | 0 | ✓ |
| / total time (seconds) | 2.104 | 2.062 | -0.04 |
| / avg req/sec | 1188.25 | 1212.63 | +24.38 |
| /error-in-render failed reqs | 0 | 0 | ✓ |
| /error-in-render total time (seconds) | 1.341 | 1.261 | -0.08 |
| /error-in-render avg req/sec | 1863.8 | 1982.97 | +119.17 |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | rasmusjp/next.js add-umbraco-heartcore-blog-example | Change |
| - | - | - | - |
| 677f882d2ed8..43e3.js gzip | 13.1 kB | 13.1 kB | ✓ |
| framework.HASH.js gzip | 39 kB | 39 kB | ✓ |
| main-7173c7a..5604.js gzip | 6.63 kB | 6.63 kB | ✓ |
| webpack-50be..df5b.js gzip | 751 B | 751 B | ✓ |
| Overall change | 59.4 kB | 59.4 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | rasmusjp/next.js add-umbraco-heartcore-blog-example | Change |
| - | - | - | - |
| polyfills-af..9390.js gzip | 31.3 kB | 31.3 kB | ✓ |
| Overall change | 31.3 kB | 31.3 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | vercel/next.js canary  | rasmusjp/next.js add-umbraco-heartcore-blog-example | Change |
| - | - | - | - |
| _app-2a09aa2..4a98.js gzip | 1.28 kB | 1.28 kB | ✓ |
| _error-8b758..aef6.js gzip | 3.46 kB | 3.46 kB | ✓ |
| hooks-c71ae4..70cd.js gzip | 887 B | 887 B | ✓ |
| index-bbee2f..528b.js gzip | 227 B | 227 B | ✓ |
| link-7faf09b..eba4.js gzip | 1.64 kB | 1.64 kB | ✓ |
| routerDirect..bf84.js gzip | 303 B | 303 B | ✓ |
| withRouter-a..5826.js gzip | 302 B | 302 B | ✓ |
| Overall change | 8.09 kB | 8.09 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | rasmusjp/next.js add-umbraco-heartcore-blog-example | Change |
| - | - | - | - |
| _buildManifest.js gzip | 321 B | 321 B | ✓ |
| Overall change | 321 B | 321 B | ✓ |

</details>

<details>
<summary><strong>Rendered Page Sizes</strong></summary>

|  | vercel/next.js canary  | rasmusjp/next.js add-umbraco-heartcore-blog-example | Change |
| - | - | - | - |
| index.html gzip | 616 B | 616 B | ✓ |
| link.html gzip | 621 B | 621 B | ✓ |
| withRouter.html gzip | 609 B | 609 B | ✓ |
| Overall change | 1.85 kB | 1.85 kB | ✓ |

</details>

</details>
<hr/>
<details>
<summary><strong>Serverless Mode</strong></summary>

<br/>

<details>
<summary><strong>General</strong></summary>

|  | vercel/next.js canary  | rasmusjp/next.js add-umbraco-heartcore-blog-example | Change |
| - | - | - | - |
| buildDuration | 12.7s | 12.9s | ⚠️ +139ms |
| nodeModulesSize | 73.3 MB | 73.3 MB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | rasmusjp/next.js add-umbraco-heartcore-blog-example | Change |
| - | - | - | - |
| 677f882d2ed8..43e3.js gzip | 13.1 kB | 13.1 kB | ✓ |
| framework.HASH.js gzip | 39 kB | 39 kB | ✓ |
| main-7173c7a..5604.js gzip | 6.63 kB | 6.63 kB | ✓ |
| webpack-50be..df5b.js gzip | 751 B | 751 B | ✓ |
| Overall change | 59.4 kB | 59.4 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | rasmusjp/next.js add-umbraco-heartcore-blog-example | Change |
| - | - | - | - |
| polyfills-af..9390.js gzip | 31.3 kB | 31.3 kB | ✓ |
| Overall change | 31.3 kB | 31.3 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | vercel/next.js canary  | rasmusjp/next.js add-umbraco-heartcore-blog-example | Change |
| - | - | - | - |
| _app-2a09aa2..4a98.js gzip | 1.28 kB | 1.28 kB | ✓ |
| _error-8b758..aef6.js gzip | 3.46 kB | 3.46 kB | ✓ |
| hooks-c71ae4..70cd.js gzip | 887 B | 887 B | ✓ |
| index-bbee2f..528b.js gzip | 227 B | 227 B | ✓ |
| link-7faf09b..eba4.js gzip | 1.64 kB | 1.64 kB | ✓ |
| routerDirect..bf84.js gzip | 303 B | 303 B | ✓ |
| withRouter-a..5826.js gzip | 302 B | 302 B | ✓ |
| Overall change | 8.09 kB | 8.09 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | rasmusjp/next.js add-umbraco-heartcore-blog-example | Change |
| - | - | - | - |
| _buildManifest.js gzip | 321 B | 321 B | ✓ |
| Overall change | 321 B | 321 B | ✓ |

</details>

<details>
<summary><strong>Serverless bundles</strong></summary>

|  | vercel/next.js canary  | rasmusjp/next.js add-umbraco-heartcore-blog-example | Change |
| - | - | - | - |
| _error.js | 1 MB | 1 MB | ✓ |
| 404.html | 2.67 kB | 2.67 kB | ✓ |
| hooks.html | 1.92 kB | 1.92 kB | ✓ |
| index.js | 1 MB | 1 MB | ✓ |
| link.js | 1.06 MB | 1.06 MB | ✓ |
| routerDirect.js | 1.05 MB | 1.05 MB | ✓ |
| withRouter.js | 1.05 MB | 1.05 MB | ✓ |
| Overall change | 5.19 MB | 5.19 MB | ✓ |

</details>

</details>
Commit: dfca77cf1996771c2e72ab2c7d8b5acedf9458e2

**Comment by ijjk** [comment]
# Stats from current PR

<details>
<summary><strong>Default Server Mode</strong> (Decrease detected ✓)</summary>

<br/>

<details>
<summary><strong>General</strong></summary>

|  | vercel/next.js canary  | rasmusjp/next.js add-umbraco-heartcore-blog-example | Change |
| - | - | - | - |
| buildDuration | 9.2s | 8.1s | -1.1s |
| nodeModulesSize | 73.3 MB | 73.3 MB | ✓ |

</details>

<details>
<summary><strong>Page Load Tests</strong> Overall decrease ⚠️</summary>

|  | vercel/next.js canary  | rasmusjp/next.js add-umbraco-heartcore-blog-example | Change |
| - | - | - | - |
| / failed reqs | 0 | 0 | ✓ |
| / total time (seconds) | 1.627 | 1.66 | ⚠️ +0.03 |
| / avg req/sec | 1537.04 | 1506.46 | ⚠️ -30.58 |
| /error-in-render failed reqs | 0 | 0 | ✓ |
| /error-in-render total time (seconds) | 1.048 | 1.096 | ⚠️ +0.05 |
| /error-in-render avg req/sec | 2385.57 | 2280.82 | ⚠️ -104.75 |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | rasmusjp/next.js add-umbraco-heartcore-blog-example | Change |
| - | - | - | - |
| 677f882d2ed8..43e3.js gzip | 13.1 kB | 13.1 kB | ✓ |
| framework.HASH.js gzip | 39 kB | 39 kB | ✓ |
| main-7173c7a..5604.js gzip | 6.63 kB | 6.63 kB | ✓ |
| webpack-50be..df5b.js gzip | 751 B | 751 B | ✓ |
| Overall change | 59.4 kB | 59.4 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | rasmusjp/next.js add-umbraco-heartcore-blog-example | Change |
| - | - | - | - |
| polyfills-af..9390.js gzip | 31.3 kB | 31.3 kB | ✓ |
| Overall change | 31.3 kB | 31.3 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | vercel/next.js canary  | rasmusjp/next.js add-umbraco-heartcore-blog-example | Change |
| - | - | - | - |
| _app-2a09aa2..4a98.js gzip | 1.28 kB | 1.28 kB | ✓ |
| _error-8b758..aef6.js gzip | 3.46 kB | 3.46 kB | ✓ |
| hooks-c71ae4..70cd.js gzip | 887 B | 887 B | ✓ |
| index-bbee2f..528b.js gzip | 227 B | 227 B | ✓ |
| link-7faf09b..eba4.js gzip | 1.64 kB | 1.64 kB | ✓ |
| routerDirect..bf84.js gzip | 303 B | 303 B | ✓ |
| withRouter-a..5826.js gzip | 302 B | 302 B | ✓ |
| Overall change | 8.09 kB | 8.09 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | rasmusjp/next.js add-umbraco-heartcore-blog-example | Change |
| - | - | - | - |
| _buildManifest.js gzip | 321 B | 321 B | ✓ |
| Overall change | 321 B | 321 B | ✓ |

</details>

<details>
<summary><strong>Rendered Page Sizes</strong></summary>

|  | vercel/next.js canary  | rasmusjp/next.js add-umbraco-heartcore-blog-example | Change |
| - | - | - | - |
| index.html gzip | 616 B | 616 B | ✓ |
| link.html gzip | 621 B | 621 B | ✓ |
| withRouter.html gzip | 609 B | 609 B | ✓ |
| Overall change | 1.85 kB | 1.85 kB | ✓ |

</details>

</details>
<hr/>
<details>
<summary><strong>Serverless Mode</strong></summary>

<br/>

<details>
<summary><strong>General</strong></summary>

|  | vercel/next.js canary  | rasmusjp/next.js add-umbraco-heartcore-blog-example | Change |
| - | - | - | - |
| buildDuration | 10.5s | 10.4s | -106ms |
| nodeModulesSize | 73.3 MB | 73.3 MB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | rasmusjp/next.js add-umbraco-heartcore-blog-example | Change |
| - | - | - | - |
| 677f882d2ed8..43e3.js gzip | 13.1 kB | 13.1 kB | ✓ |
| framework.HASH.js gzip | 39 kB | 39 kB | ✓ |
| main-7173c7a..5604.js gzip | 6.63 kB | 6.63 kB | ✓ |
| webpack-50be..df5b.js gzip | 751 B | 751 B | ✓ |
| Overall change | 59.4 kB | 59.4 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | rasmusjp/next.js add-umbraco-heartcore-blog-example | Change |
| - | - | - | - |
| polyfills-af..9390.js gzip | 31.3 kB | 31.3 kB | ✓ |
| Overall change | 31.3 kB | 31.3 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | vercel/next.js canary  | rasmusjp/next.js add-umbraco-heartcore-blog-example | Change |
| - | - | - | - |
| _app-2a09aa2..4a98.js gzip | 1.28 kB | 1.28 kB | ✓ |
| _error-8b758..aef6.js gzip | 3.46 kB | 3.46 kB | ✓ |
| hooks-c71ae4..70cd.js gzip | 887 B | 887 B | ✓ |
| index-bbee2f..528b.js gzip | 227 B | 227 B | ✓ |
| link-7faf09b..eba4.js gzip | 1.64 kB | 1.64 kB | ✓ |
| routerDirect..bf84.js gzip | 303 B | 303 B | ✓ |
| withRouter-a..5826.js gzip | 302 B | 302 B | ✓ |
| Overall change | 8.09 kB | 8.09 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | rasmusjp/next.js add-umbraco-heartcore-blog-example | Change |
| - | - | - | - |
| _buildManifest.js gzip | 321 B | 321 B | ✓ |
| Overall change | 321 B | 321 B | ✓ |

</details>

<details>
<summary><strong>Serverless bundles</strong></summary>

|  | vercel/next.js canary  | rasmusjp/next.js add-umbraco-heartcore-blog-example | Change |
| - | - | - | - |
| _error.js | 1 MB | 1 MB | ✓ |
| 404.html | 2.67 kB | 2.67 kB | ✓ |
| hooks.html | 1.92 kB | 1.92 kB | ✓ |
| index.js | 1 MB | 1 MB | ✓ |
| link.js | 1.06 MB | 1.06 MB | ✓ |
| routerDirect.js | 1.05 MB | 1.05 MB | ✓ |
| withRouter.js | 1.05 MB | 1.05 MB | ✓ |
| Overall change | 5.19 MB | 5.19 MB | ✓ |

</details>

</details>
Commit: 9eba92fb7c17534f873ec8da76e530f1291a6725

File: examples/cms-umbraco-heartcore/README.md [hunk 1]
```diff
@@ -0,0 +1,148 @@
+# A statically generated blog example using Next.js and Umbraco Heartcore
+
+This example showcases Next.js's [Static Generation](https://nextjs.org/docs/basic-features/pages) feature using [Umbraco Heartcore](https://umbraco.com/heartcore) as the data source.
+
+## Demo
+
+- **Live**: [https://next-blog-umbraco-heartcore.vercel.app/](https://next-blog-umbraco-heartcore.vercel.app/)
+- **Preview Mode**: [https://next-blog-umbraco-heartcore.vercel.app/api/preview...](https://next-blog-umbraco-heartcore.vercel.app/api/preview?secret=ivfgc53v8d8ikh7z4nnd5hzwgs298ma6&slug=/posts/learn-how-to-pre-render-pages-using-static-generation-with-next-js)
```
Comment (leerob) [inline examples/cms-umbraco-heartcore/README.md:8]: Is it okay to include this?

File: examples/cms-umbraco-heartcore/README.md [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+
+```bash
+npx create-next-app --example cms-umbraco-heartcore cms-umbraco-heartcore-app
+# or
+yarn create next-app --example cms-umbraco-heartcore cms-umbraco-heartcore-app
+```
+
+## Configuration
+
+### Step 1. Create an account and a project in Umbraco Heartcore
```
Comment (leerob) [inline examples/cms-umbraco-heartcore/README.md:45]: An Integration with Vercel could make this experience much easier 😄

Comment (leerob) [reply examples/cms-umbraco-heartcore/README.md:45]: e.g. https://vercel.com/virtual-event-starter-kit which has the DatoCMS integration on clone & deploy

File: examples/cms-umbraco-heartcore/components/cover-image.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
+import cn from 'classnames'
+import Link from 'next/link'
+
+export default function CoverImage({ title, url, slug }) {
+  const image = (
+    <img
```
Comment (leerob) [inline examples/cms-umbraco-heartcore/components/cover-image.js:6]: This should use next/image 😄

File: examples/cms-umbraco-heartcore/components/cover-image.js [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+        'hover:shadow-medium transition-shadow duration-200': slug,
+      })}
+      src={url}
+    />
+  )
+
+  return (
+    <div className="sm:mx-0">
+      {slug ? (
+        <Link as={slug} href="[...slug]">
```
Comment (leerob) [inline examples/cms-umbraco-heartcore/components/cover-image.js:20]: `as` isn't needed anymore.

```suggestion
        <Link href={slug}>
```

File: examples/cms-umbraco-heartcore/components/hero-post.js [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+}) {
+  return (
+    <section>
+      <div className="mb-8 md:mb-16">
+        <CoverImage slug={slug} title={title} url={coverImage.url} />
+      </div>
+      <div className="mb-20 md:grid md:grid-cols-2 md:col-gap-16 lg:col-gap-8 md:mb-28">
+        <div>
+          <h3 className="mb-4 text-4xl leading-tight lg:text-6xl">
+            <Link as={slug} href="[...slug]">
```
Comment (leerob) [inline examples/cms-umbraco-heartcore/components/hero-post.js:22]:
```suggestion
            <Link href={slug}>
```

File: examples/cms-umbraco-heartcore/components/post-preview.js [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+  author,
+  slug,
+}) {
+  return (
+    <div>
+      <div className="mb-5">
+        <CoverImage slug={slug} title={title} url={coverImage.url} />
+      </div>
+      <h3 className="mb-3 text-3xl leading-snug">
+        <Link as={slug} href="[...slug]">
```
Comment (leerob) [inline examples/cms-umbraco-heartcore/components/post-preview.js:20]:
```suggestion
        <Link href={slug}>
```

File: examples/cms-umbraco-heartcore/lib/umbraco-heartcore.js [hunk 7]
```diff
@@ -0,0 +1,153 @@
...
+        author {
+          ...on Author {
+            name
+            picture {
+              url(width: 100, height: 100, cropMode: CROP, upscale: true)
+            }
+          }
+        }
+      }
+			morePosts: allPost(first: 2, where: { NOT: { url: $slug } }, orderBy: [date_DESC], preview: $preview) {
```
Comment (leerob) [inline examples/cms-umbraco-heartcore/lib/umbraco-heartcore.js:118]: Formatting here seems off

File: examples/cms-umbraco-heartcore/pages/[...slug].js [hunk 8]
```diff
@@ -0,0 +1,71 @@
...
+        {router.isFallback ? (
+          <PostTitle>Loading…</PostTitle>
+        ) : (
+          <>
+            <article>
+              <Head>
+                <title>
+                  {post.title} | Next.js Blog Example with {CMS_NAME}
+                </title>
+                {/* <meta property="og:image" content={post.ogImage.url} /> */}
```
Comment (leerob) [inline examples/cms-umbraco-heartcore/pages/[...slug].js:35]: Why is this commented out?

**Review by leerob — CHANGES_REQUESTED** [review]
Thank you!

File: examples/cms-umbraco-heartcore/README.md (hunk 1 above)
Comment (rasmusjp) [reply examples/cms-umbraco-heartcore/README.md:8]: Should be fine as it only gives access to preview, I looked at the GraphCMS example and theres also have a link to preview mode

File: examples/cms-umbraco-heartcore/components/cover-image.js (hunk 3 above)
Comment (rasmusjp) [reply examples/cms-umbraco-heartcore/components/cover-image.js]: updated to use next/image

File: examples/cms-umbraco-heartcore/components/cover-image.js (hunk 4 above)
Comment (rasmusjp) [reply examples/cms-umbraco-heartcore/components/cover-image.js]: updated

File: examples/cms-umbraco-heartcore/components/hero-post.js (hunk 5 above)
Comment (rasmusjp) [reply examples/cms-umbraco-heartcore/components/hero-post.js]: updated

File: examples/cms-umbraco-heartcore/lib/umbraco-heartcore.js (hunk 7 above)
Comment (rasmusjp) [reply examples/cms-umbraco-heartcore/lib/umbraco-heartcore.js]: some tabs had sneaked in, fixed now

File: examples/cms-umbraco-heartcore/pages/[...slug].js (hunk 8 above)
Comment (rasmusjp) [reply examples/cms-umbraco-heartcore/pages/[...slug].js]: Not, sure. Uncommented now 😄

**Comment by rasmusjp** [comment]
Hi, do you need anything else from me?

**Comment by leerob** [comment]
@rasmusjp 👋 Sorry for the slow response, this one slipped through. Would you mind addressing merge conflicts and I will give it another review? Thank you!

**Comment by rasmusjp** [comment]
@leerob No worries, I've updated with the latest from `canary`, also updated all the dependencies to latest versions

File: examples/cms-umbraco-heartcore/components/avatar.js [hunk 9]
```diff
@@ -0,0 +1,8 @@
+export default function Avatar({ name, picture }) {
+  return (
+    <div className="flex items-center">
+      <img src={picture} className="w-12 h-12 rounded-full mr-4" alt={name} />
```
Comment (leerob) [inline examples/cms-umbraco-heartcore/components/avatar.js:4]: Could we use `next/image`? Some of the other CMS examples will show this!

File: examples/cms-umbraco-heartcore/components/cover-image.js [hunk 10]
```diff
@@ -0,0 +1,29 @@
+import cn from 'classnames'
+import Link from 'next/link'
+import Image from 'next/image'
+
+export default function CoverImage({ title, url, slug }) {
+  const image = (
+    <Image
```
Comment (leerob) [inline examples/cms-umbraco-heartcore/components/cover-image.js:7]: Like here! 😄

File: examples/cms-umbraco-heartcore/components/date.js [hunk 11]
```diff
@@ -0,0 +1,6 @@
+import { parseISO, format } from 'date-fns'
+
+export default function Date({ dateString }) {
+  const date = parseISO(dateString)
+  return <time dateTime={dateString}>{format(date, 'LLLL	d, yyyy')}</time>
```
Comment (leerob) [inline examples/cms-umbraco-heartcore/components/date.js:5]: Is the formatting off here?

File: examples/cms-umbraco-heartcore/package.json [hunk 12]
```diff
@@ -0,0 +1,22 @@
+{
+  "name": "cms-umbraco-heartcore",
+  "version": "1.0.0",
```
Comment (leerob) [inline examples/cms-umbraco-heartcore/package.json:3]:
```suggestion
  "private": true,
```

File: examples/cms-umbraco-heartcore/package.json [hunk 13]
```diff
@@ -0,0 +1,22 @@
...
+    "next": "latest",
+    "react": "^17.0.2",
+    "react-dom": "^17.0.2"
+  },
+  "devDependencies": {
+    "postcss-flexbugs-fixes": "^5.0.2",
+    "postcss-preset-env": "^7.0.1",
+    "tailwindcss": "^2.2.5"
+  },
+  "license": "MIT"
```
Comment (leerob) [inline examples/cms-umbraco-heartcore/package.json:21]:
```suggestion
```

File: examples/cms-umbraco-heartcore/tailwind.config.js [hunk 14]
```diff
@@ -0,0 +1,33 @@
+module.exports = {
+  purge: ['./components/**/*.js', './pages/**/*.js'],
```
Comment (leerob) [inline examples/cms-umbraco-heartcore/tailwind.config.js:2]: Can we add JIT mode please?

File: examples/cms-umbraco-heartcore/components/avatar.js (hunk 9 above)
Comment (rasmusjp) [reply examples/cms-umbraco-heartcore/components/avatar.js]: Updated

File: examples/cms-umbraco-heartcore/components/date.js (hunk 11 above)
Comment (rasmusjp) [reply examples/cms-umbraco-heartcore/components/date.js]: Looks like there were a tab, fixed now

File: examples/cms-umbraco-heartcore/tailwind.config.js (hunk 14 above)
Comment (rasmusjp) [reply examples/cms-umbraco-heartcore/tailwind.config.js]: Updated config to match the other examples

File: examples/cms-umbraco-heartcore/package.json [hunk 15]
```diff
@@ -1,6 +1,7 @@
 {
   "name": "cms-umbraco-heartcore",
   "version": "1.0.0",
```
Comment (leerob) [inline examples/cms-umbraco-heartcore/package.json]:
```suggestion
```

Comment (leerob) [reply examples/cms-umbraco-heartcore/package.json]: Nvm, this is outdated

**Review by leerob — DISMISSED** [review]

File: examples/cms-umbraco-heartcore/package.json (hunk 12 above)
Comment (leerob) [inline examples/cms-umbraco-heartcore/package.json:3]:
```suggestion
```

**Review by leerob — DISMISSED** [review]

**Review by timneutkens — APPROVED** [review]
Re-approving based on @leerob's approve. Didn't review the PR.

**Pull request merged**